### PR TITLE
Add support for typical phone number formats used in the Netherlands, Italy, and Denmark

### PR DESCRIPTION
**Pull request by godlovesun**
#778

### CODE REVIEW

File: .NET/Microsoft.Recognizers.Definitions/BasePhoneNumbers.cs [hunk 1]
```diff
@@ -24,6 +24,9 @@ public static class BasePhoneNumbers
 		public const string GermanyPhoneNumberRegex = @"(\b(\d{3,4}\s?[-/]?(\s?\d){7,8})\b)";
 		public const string USPhoneNumberRegex = @"((((\B\+)|\b)1(\s|-)?)|\b)?(\(\d{3}\)|\b\d{3})\s?-?\s?\d{3}\s?-?\s?\d{4}\b";
 		public const string CNPhoneNumberRegex = @"((\b00\s?)?\+?86\s?-?\s?)?((\d{2,5}|\(\d{2,5}\))\s?-?\s?\d{4}\s?-?\s?\d{4}(\s?-?\s?\d{4})?|\d{3}\s?-?\s?\d{4}\s?-?\s?\d{4})(?!-)\b";
+		public const string DenmarkPhoneNumberRegex = @"((\(\s?(\+\s?|00)45\s?\)(\s|-)?)|(((\B\+)|\b00|\b)\s?45(\s|-)?)|\b)((\d{8})|(\d{4}\s?-?\s?\d{4})|((\d{2}[\s-]){3}\d{2})|(\d{2}\s?-?\s?\d{3}\s?-?\s?\d{3}))\b";
```
[the remark this answers is not in the document]
Comment (godlovesun) [reply .NET/Microsoft.Recognizers.Definitions/BasePhoneNumbers.cs]: This issue has been updated in latest commit. And the exsited reges has alse been unformed.

File: .NET/Microsoft.Recognizers.Text.Sequence/Constants.cs [hunk 2]
```diff
@@ -37,16 +37,22 @@ public static class Constants
...
         public const string PHONE_NUMBER_REGEX_US = "USPhoneNumber";
 
         public const string PHONE_NUMBER_REGEX_CN = "CNPhoneNumber";
-        
+
+        public const string PHONE_NUMBER_REGEX_DK = "DKPhoneNumber";
+
+        public const string PHONE_NUMBER_REGEX_IT = "ITPhoneNumber";
+
+        public const string PHONE_NUMBER_REGEX_NL = "NLlandsPhoneNumber";
```
Comment (iMicknl) [inline .NET/Microsoft.Recognizers.Text.Sequence/Constants.cs]: `NLPhoneNumber` would be fine for the Netherlands. (instead of `NLlandsPhoneNumber`)

Comment (godlovesun) [reply .NET/Microsoft.Recognizers.Text.Sequence/Constants.cs]: Yeah, sorry, that is a mistake, I will fix that, thanks.

Comment (godlovesun) [reply .NET/Microsoft.Recognizers.Text.Sequence/Constants.cs]: Our model work well with these two cases and the cases have been added to cases, thanks for providing them!

**Pull request merged**